examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -187,6 +216,11 @@ const uint32_t kMqttReconnectTime = 5000;  // Delay(ms) between reconnect tries.
 #define MQTTrecv MQTTprefix "/received"  // Topic we send received IRs to.
 #endif  // MQTT_ENABLE
 
+#if (FIRMWARE_OTA_PASSWORD_ENABLE || HTML_PASSWORD_ENABLE)
+const char* kHtmlUsername = "admin";    // <=- CHANGE_ME (optional)
+const char* kHtmlPassword = "esp8266";  // <=- CHANGE_ME
```

Comment:
Just an idea, but I'm a bit worried about people not changing the password. What if by default it would make the build fail / or it would generate a random password on each run (making it unusable until it is changed)? Just an idea as if the users doesn't change the password, it is on them.
Although this might be tricky to test automatically.

Reply:
I don't like the idea of a _default_ password either, but for now, this solves the immediate problem.

Setting an assert so it fails on compile if it hasn't been changed is problematic for the automated builds. Workarounds for that could be done but ...

Another option is requiring a user set a password on initial (wifi) setup. That gets rid of the _default password_ issue, but leads us back to the initial problem of an attacker gaining access to the device via blocking wifi access etc. i.e. They'll be able to set a new password and then upload firmware.

Generating a random (or derived from site variables. e.g. ip, ssid, wifipassword, mqttserver or chipid) is that we'd need to display it to the user some how, and we'd be almost in the same situation as it could be derived by an attacker.

An option/solution could be to disable OTA firmware upload *if* the firmware password matches the _default_ password. That's about the easiest/simplest/best solution I can think of. I'll look at that for a later revision.